todo_app/app.py

```diff
@@ -10,90 +10,58 @@
 
 KEY=os.getenv("KEY")
 TOKEN=os.getenv("TOKEN")
-    
 
-class TransportAPI:
-    def __init__(self):
-        api_keys = get_application_keys()
-        self.payload = api_keys
+BOARD_ID=os.getenv("BOARD_ID")
+ID_LIST_TODO=os.getenv("ID_LIST_TODO")
+ID_LIST_DOING=os.getenv("ID_LIST_DOING")
+ID_LIST_DONE=os.getenv("ID_LIST_DONE")
 
 
+class Item:
+   
+   def __init__(self, id, title, status):
+      self.title = title
+      self.status = status
+      self.id = id  
+
 @app.route('/index')
```

Comment:
Not related to latest changes but I'd suggest leaving this url as just `/` rather than `/index` - having your web domain error 404 on it's root page can be very confusing!